### DISCUSSION TITLE
Untranslated "What's Next?"

### DISCUSSION THREAD

**Discussion by garak** [symfony/symfony#50938] [Symfony Book: The Fast Track]
The "What's Next?" page has English text in all the translations (except for the final sentence). Most of them are fully translated, so it's some kind of bug.

See:
https://symfony.com/doc/6.2/the-fast-track/fr/next.html
https://symfony.com/doc/6.2/the-fast-track/de/next.html
https://symfony.com/doc/6.2/the-fast-track/it/next.html

etc...

**Comment by lyrixx** [comment]
I cannot find this sentence on the pages you linked

![image](https://github.com/symfony/symfony/assets/408368/a2e3ce64-78f7-402b-9e83-604ac3fc571d)

**Comment by garak** [comment]
The sentence is the one starting with "I hope you..."

**Reply by lyrixx** [reply]
Ok, I miss understood the issue. Sorry!
